roles/backend_setup/tasks/bind_tang_server.yml

```diff
@@ -19,23 +19,54 @@
   when: gluster_infra_tangservers is defined
 
 - name: Bind tang server with clevis
+  no_log: true
   shell: clevis luks bind -f -k /etc/root_key -d  {{ rootdevice }} tang '{"url":"{{item.url}}", "adv":"/etc/adv{{ index }}.jws"}'
   loop: "{{ gluster_infra_tangservers }}"
   loop_control:
     index_var: index
   when: gluster_infra_tangservers is defined
 
-# TODO Need to investigate how to configure incase of static deployment
-- name: Add network entry on /etc/dracut.conf.d/clevis.conf
+# For DHCP
+- name: Add network entry for dhcp on /etc/dracut.conf.d/clevis.conf
   blockinfile:
     path: /etc/dracut.conf.d/clevis.conf
     create: yes
     state: present
     block: |
       kernel_cmdline="ip={{ networkinterface }}:dhcp"
       omit_dracutmodules+="ifcfg"
+      omit_dracutmodules+="network-legacy"
+      add_dracutmodules+="clevis network-manager"
```

Comment:
Hey @gobindadas ,is the space expected between clevis and network-manager?

Reply:
yes

Reply:
@gobindadas The syntax is incorrect for IPv6, format shold be:
kernel_cmdline="ip={{[host_ip_addr]}}::{{[host_net_gateway]}}:{{host_ip_prefix}}::{{interface}}:off"